cloud/amazon/rds.py

```diff
@@ -647,8 +652,9 @@ def get_data(self):
 
 def await_resource(conn, resource, status, module):
     wait_timeout = module.params.get('wait_timeout') + time.time()
+    api_throttling = module.params.get('api_throttling')
```

Comment:
@ryansb I'd love to dig more about the AWSRetry. But for the simplicity of solving the api throttling issue, can we have this as a quick fix?